index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```

Comment:
Hi @Jolak5 

Well done on the work done so far! You're superb. 

You still have a few works left to go though